### ISSUE TITLE
Transform with filter in transitions

### ISSUE THREAD

**Issue by pieter2627** [postcss/autoprefixer#551]
When you have a `transform` together with `filter` on the `transition` property then then there ends up being two `-webkit-transform`'s where the second should just be `transform` like the second `filter`.

Example using the online tool:
### Input
```
.example {
    transition-property: transform, filter;
}
```

### Output
```
.example {
    -webkit-transition-property: -webkit-transform, -webkit-filter, -webkit-transform, filter;
            transition-property: transform, filter;
            transition-property: -webkit-transform, -webkit-filter, -webkit-transform, filter;
}
```

### Result
Browser that has support without the use of prefixes does not have any transitions on the transform as the 3rd line overwrote the 2nd and the 3rd does not have the correct transform.

PS. replacing filter with any other value does not result in the 3rd line

**Comment by ai** [comment]
This error is fixed in `master` and will be released in 6.1 on this week. Current output:

```css
.example {
    -webkit-transition-property: -webkit-transform, -webkit-filter;
    transition-property: -webkit-transform, -webkit-filter;
    transition-property: transform, filter;
    transition-property: transform, filter, -webkit-transform, -webkit-filter;
}
```

You can subscribe to [@autoprefixer](https://twitter.com/autoprefixer) and [@postcss](https://twitter.com/postcss) for futher next release announces :).

**Comment by pieter2627** [comment]
Great :thumbsup:
